talentsearch/webpack.mix.js

```diff
@@ -65,3 +65,4 @@ mix.webpackConfig({
 });
 
 mix.version();
+mix.sourceMaps();
```

Comment:
I'm not sure you should include this change in your PR.  This is useful on your own machine for testing but is not relevant to the issue you're working on.

Reply:
without this we wouldn't be able to debug. So I thought it should be in repo. However I'm unsure if this has any negative impact .. so put it up for suggestions. 
Yeah ... definitely not related to this issue.